.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,57 @@
+<!-- Thank you for submitting a pull request to the HPCC project
+
+ PLEASE READ the following before proceeding.
+
+ Lines in comments may be deleted from the comment before you submit.
+ Other lines should be modified appropriately and left in place.
+
+ This project only accepts pull requests related to open JIRA issues.
+ If suggesting a new feature or change, please discuss it in a JIRA issue first.
+ If fixing a bug, there should be an issue describing it with steps to reproduce.
+ The title line of the pull request (and of each commit within it) should refer to the
+ associated issue using the format:
+
+ HPCC-nnnnn Short description of issue
+
+ This will allow the Jira ticket to be automatically updated to refer to this pull request,and
+ and will ensure that the automatically-generated changelog is properly formatted.
+ Where a pull request contains a single commit the pull request title will be set automatically,
+ assuming that the commit has followed the proper guidelines.
+
+ Please go over all the following points, and put an `x` in all the boxes that apply. You may find
+ it easier to press the 'Create' button first then click on the checkboxes to edit the comment.
+-->
+
+## Type of change:
+- [ ] This change is a bug fix (non-breaking change which fixes an issue).
+- [ ] This change is a new feature (non-breaking change which adds functionality).
```

Comment:
Do you also want an option for a change that breaks deployed queries, but doesn't change their behaviour?